apis/excel/tool.gpt

```diff
@@ -65,4 +65,12 @@ Type: context
 You have access to a set of tools to access, create, and modify Microsoft Excel workbooks.
 Do not output workbook IDs or worksheet IDs because they are not helpful for the user.
 
-## End of instructions for using Excel tools
\ No newline at end of file
+## End of instructions for using Excel tools
+
+---
+!metadata:*:category
+Excel
+
+---
+!metadata:*:icon
+https://raw.githubusercontent.com/phosphor-icons/core/refs/tags/v2.0.8/raw/regular/microsoft-excel-logo.svg
```

Comment:
```suggestion
https://cdn.jsdelivr.net/npm/@phosphor-icons/core@2/assets/regular/microsoft-excel-logo.svg
```

Reply:
Alternatively to be consistent with the other phosphor icons being used that are duotone - 
```suggestion
https://cdn.jsdelivr.net/npm/@phosphor-icons/core@2/assets/duotone/microsoft-excel-logo-duotone.svg
```